platform/web/os_web.cpp

```diff
@@ -152,10 +152,12 @@ bool OS_Web::_check_internal_feature_support(const String &p_feature) {
 	if (p_feature == "web") {
 		return true;
 	}
-	if (godot_js_os_has_feature(p_feature.utf8().get_data())) {
-		return true;
+	if (feature_support_cache.has(p_feature)) {
+		return feature_support_cache[p_feature];
 	}
```

Comment:
You can avoid accessing the map twice using `find()`. Not sure it will make a big difference though since the map will contain few elements.
```suggestion
	HashMap<String, bool>::Iterator it = feature_support_cache.find(p_feature);
	if (it != feature_support_cache.end()) {
		return it->value;
	}
```